site/management.md

```diff
@@ -373,6 +378,8 @@ such as:
   * <*resource_server_id*>`.tag:administrator`
   * <*resource_server_id*>`.read:*/*/*`
 
+We use the setting `management.oauth_scopes` to configure the scopes. It is a space-separated field.
+
```

Comment:
Will a customer/user know what <It is a space-separated field.> My understanding is you enter a scope followed by a space so: scope1 scope2 scope3

If that is correct then there probably is no need to clarify further, if I can understand more technical people definitely should :)

Reply:
That is !! :)